docs/references/config/config.toml.md

```diff
@@ -0,0 +1,1749 @@
+|:----------------------------------------|:----------------|
+| **Possible string values in the array** | `"HEAD"`        |
+|                                         | `"GET"`         |
+|                                         | `"POST"`        |
+
+You can read more about the methods in the
+[Mozilla CORS documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS).
+
+### rpc.cors_allowed_headers
+A list of headers the client is allowed to use with cross-domain requests.
+```toml
+cors_allowed_headers = ["Origin", "Accept", "Content-Type", "X-Requested-With", "X-Server-Time", ]
+```
+
+| Value type                              | array of string      |
+|:----------------------------------------|:---------------------|
+| **Possible string values in the array** | `"Accept"`           |
+|                                         | `"Accept-Language"`  |
+|                                         | `"Content-Language"` |
+|                                         | `"Content-Type"`     |
+|                                         | `"Range"`            |
+
+The list of possible values are from the [Fetch spec](https://fetch.spec.whatwg.org/#cors-safelisted-request-header)
+which defines `Origin` as a forbidden value. Read the
+[Mozilla CORS documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) and do your own tests, if you want
+to use this key.
+
+<!--- Possibly, we should clarify the allowed values better. --->
+
+### rpc.unsafe
+Activate unsafe RPC endpoints.
+```toml
+unsafe = false
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `false` |
+|                     | `true`  |
+
+| Unsafe RPC endpoints    | Description                                                                           |
+|:------------------------|---------------------------------------------------------------------------------------|
+| `/dial_seeds`           | dials the given seeds (comma-separated id@IP:port)                                    |
+| `/dial_peers`           | dials the given peers (comma-separated id@IP:port), optionally making them persistent |
+| `/unsafe_flush_mempool` | removes all transactions from the mempool                                             |
+
+Keep this `false` on production systems.
+
+### rpc.max_open_connections
+Maximum number of simultaneous open connections. This includes WebSocket connections.
+```toml
+max_open_connections = 900
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt; 0  |
+
+If you want to accept a larger number of connections than the default 900, make sure that you increase the maximum
+number of open connections in the operating system. Usually, the `ulimit` command can help with that.
+
+This value can be estimated by the following calculation:
+```
+$(ulimit -Sn) - {p2p.max_num_inbound_peers} - {p2p.max_num_outbound_peers} - {number of WAL, DB and other open files}
```

Comment:
We should also add the persistent peers, but this will become super complex.

Reply:
I think we shouldn't get to overly complex here. I think this is good enough for the config. If we need to provide more insights on parameter calculations or estimations, we should do that on a separate guide for operators.